--- sox-12.17.6/src/Makefile.in.orig	2004-10-03 03:02:24.000000000 +0200
+++ sox-12.17.6/src/Makefile.in	2004-10-21 22:01:10.089196688 +0200
@@ -102,24 +102,24 @@
 PLAY_INSTALL_1    = install-play
 
 install: sox soxmix $(PLAY_INSTALL_$(PLAY_SUPPORT))
-	$(top_srcdir)/mkinstalldirs $(bindir)
-	$(INSTALL) -c -m 755 sox $(bindir)
-	$(INSTALL) -c -m 755 soxmix $(bindir)
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 755 sox $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 755 soxmix $(DESTDIR)$(bindir)
 
 install-play:
-	if [ -f $(bindir)/rec ] ; then $(RM) $(bindir)/rec; fi
-	$(top_srcdir)/mkinstalldirs $(bindir)
-	$(INSTALL) -c -m 755 play $(bindir)
-	$(LN_S) $(bindir)/play $(bindir)/rec
+	if [ -f $(DESTDIR)$(bindir)/rec ] ; then $(RM) $(DESTDIR)$(bindir)/rec; fi
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 755 play $(DESTDIR)$(bindir)
+	$(LN_S) $(bindir)/play $(DESTDIR)$(bindir)/rec
 
 install-lib: libst.a
-	$(top_srcdir)/mkinstalldirs $(libdir)
-	$(top_srcdir)/mkinstalldirs $(includedir)
-	$(INSTALL) -c -m 755 libst-config $(bindir)
-	$(INSTALL) -c -m 644 libst.a $(libdir)
-	$(RANLIB) $(libdir)/libst.a
-	$(INSTALL) -c -m 644 $(srcdir)/st.h $(includedir)
-	$(INSTALL) -c -m 644 ststdint.h $(includedir)
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(libdir)
+	$(top_srcdir)/mkinstalldirs $(DESTDIR)$(includedir)
+	$(INSTALL) -c -m 755 libst-config $(DESTDIR)$(bindir)
+	$(INSTALL) -c -m 644 libst.a $(DESTDIR)$(libdir)
+	$(RANLIB) $(DESTDIR)$(libdir)/libst.a
+	$(INSTALL) -c -m 644 $(srcdir)/st.h $(DESTDIR)$(includedir)
+	$(INSTALL) -c -m 644 ststdint.h $(DESTDIR)$(includedir)
 
 PLAY_UNINSTALL_0    =
 PLAY_UNINSTALL_1    = uninstall-play
--- sox-12.17.6/Makefile.in.orig	2004-09-02 00:04:02.000000000 +0200
+++ sox-12.17.6/Makefile.in	2004-10-21 21:58:35.873641016 +0200
@@ -29,22 +29,22 @@
 
 install: $(PLAY_INSTALL_$(PLAY_SUPPORT))
 	cd src && $(MAKE) $@
-	$(srcdir)/mkinstalldirs $(mandir)/man1
-	$(INSTALL) -c -m 644 $(srcdir)/sox.1 $(mandir)/man1
-	if [ -f $(mandir)/man1/soxmix.1 ] ; then $(RM) $(mandir)/man1/soxmix.1; fi
-	$(LN_S) $(mandir)/man1/sox.1 $(mandir)/man1/soxmix.1
-	$(INSTALL) -c -m 644 $(srcdir)/soxexam.1 $(mandir)/man1
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -c -m 644 $(srcdir)/sox.1 $(DESTDIR)$(mandir)/man1
+	if [ -f $(DESTDIR)$(mandir)/man1/soxmix.1 ] ; then $(RM) $(DESTDIR)$(mandir)/man1/soxmix.1; fi
+	echo '.so sox.1' > $(DESTDIR)$(mandir)/man1/soxmix.1
+	$(INSTALL) -c -m 644 $(srcdir)/soxexam.1 $(DESTDIR)$(mandir)/man1
 
 install-play:
-	if [ -f $(mandir)/man1/rec.1 ]; then $(RM) $(mandir)/man1/rec.1; fi
-	$(srcdir)/mkinstalldirs $(mandir)/man1
-	$(INSTALL) -c -m 644 $(srcdir)/play.1 $(mandir)/man1
-	$(LN_S) $(mandir)/man1/play.1 $(mandir)/man1/rec.1
+	if [ -f $(DESTDIR)$(mandir)/man1/rec.1 ]; then $(RM) $(DESTDIR)$(mandir)/man1/rec.1; fi
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/man1
+	$(INSTALL) -c -m 644 $(srcdir)/play.1 $(DESTDIR)$(mandir)/man1
+	$(LN_S) $(mandir)/man1/play.1 $(DESTDIR)$(mandir)/man1/rec.1
 
 install-lib:
 	cd src && $(MAKE) $@
-	$(srcdir)/mkinstalldirs $(mandir)/man3
-	$(INSTALL) -c -m 644 $(srcdir)/libst.3 $(mandir)/man3
+	$(srcdir)/mkinstalldirs $(DESTDIR)$(mandir)/man3
+	$(INSTALL) -c -m 644 $(srcdir)/libst.3 $(DESTDIR)$(mandir)/man3
 
 
 PLAY_UNINSTALL_0    =
