--- sox-12.15/rec.orig	Mon Nov 23 17:08:27 1998
+++ sox-12.15/rec	Mon Nov 23 17:08:42 1998
@@ -24,7 +24,7 @@
 
 # Set up path so that it can find Sox if user's path doesn't already
 # include it.
-PATH=$PATH:/usr/local/bin
+PATH=$PATH:/usr/bin
 export PATH
 
 help()
--- sox-12.15/play.orig	Sat Oct 21 12:32:37 2000
+++ sox-12.15/play	Sat Oct 21 12:34:45 2000
@@ -14,7 +14,7 @@
 
 # Set up path so that it can find Sox if user's path doesn't already
 # include it.
-PATH=$PATH:/usr/local/bin
+PATH=$PATH:/usr/bin
 program_name=`basename $0`
 program_version="2.0"
 
