--- sox-12.15/play.orig	Wed Jan 20 11:50:22 1999
+++ sox-12.15/play	Wed Jan 20 11:50:32 1999
@@ -117,6 +117,6 @@
 #    fi
 
     # Best to always use highest quality output for sound.
-    sox $volume $fopts $filename -t ossdsp -w -s /dev/dsp $effects
+    sox $volume $fopts "$filename" -t ossdsp -w -s /dev/dsp $effects
   fi
 fi
--- sox-12.15/rec.orig	Wed Jan 20 11:50:25 1999
+++ sox-12.15/rec	Wed Jan 20 11:50:40 1999
@@ -114,6 +114,6 @@
 #      mixer $volume
 #    fi
 
-    sox $volume -t ossdsp /dev/dsp $fopts $filename $effects
+    sox $volume -t ossdsp /dev/dsp $fopts "$filename" $effects
   fi
 fi
