--- sox-12.17.1/avr.c.saywhat	Tue Jan  2 11:48:05 2001
+++ sox-12.17.1/avr.c	Tue Jan  2 11:48:12 2001
@@ -38,7 +38,7 @@
   unsigned short midi; /* 0xffff = no MIDI note assigned,
 			  0xffXX = single key note assignment
 			  0xLLHH = key split, low/hi note */
-  ULONG int rate;	       /* sample frequency in hertz */
+  ULONG rate;	       /* sample frequency in hertz */
   ULONG size;	       /* sample length in bytes or words (see rez) */
   ULONG lbeg;	       /* offset to start of loop in bytes or words.
 			  set to zero if unused. */
