--- sox-12.17.7/src/play.in.orig	Sat Jan 19 21:12:49 2002
+++ sox-12.17.7/src/play.in	Sat Jan 19 21:14:47 2002
@@ -156,37 +156,10 @@
     shift
 done
 
-arch=`uname -s`
-case $arch in
-  SunOS)
-    case `uname -r` in
-        # Solaris software can auto-detect hardware capabilities.
-        5.*)
-	    arch_defines="-t sunau"
-	    ;;
-	# For SunOS default to signed words.  Some hardware can only play u-law and would need
-	# to be changed here.
-	*)
-	    arch_defines="-t sunau -w -s"
-	    ;;
-    esac
-    if [ -z "$device" ]; then
-	device="/dev/audio"
-    fi
-    ;;
-  Linux|FreeBSD)
-    arch_defines="-t ossdsp"
-    if [ -z "$device" ]; then
-	device="/dev/dsp"
-    fi
-    ;;
-  NetBSD|OpenBSD)
-    arch_defines="-t sunau"
-    if [ -z "$device" ]; then
-        device="/dev/audio"
-    fi
-    ;;
-esac
+arch_defines="-t ossdsp"
+if [ -z "$device" ]; then
+    device="/dev/dsp"
+fi
 
 # If name is "rec" then record else assume user is wanting to play
 # a sound file.
