--- sox-13.0.0/src/gsm.c.orig	2007-01-25 02:02:01.000000000 +0100
+++ sox-13.0.0/src/gsm.c	2007-02-17 21:35:51.300359570 +0100
@@ -29,7 +29,11 @@
 #include "st_i.h"
 
 #ifdef EXTERNAL_GSM
+#ifdef HAVE_GSM_H
+#include <gsm.h>
+#else
 #include <gsm/gsm.h>
+#endif
 #else
 #include "libgsm/gsm.h"
 #endif
--- sox-13.0.0/src/wav.c.orig	2007-02-06 14:45:37.000000000 +0100
+++ sox-13.0.0/src/wav.c	2007-02-17 21:36:08.057314493 +0100
@@ -24,7 +24,11 @@
 #include "ima_rw.h"
 #include "adpcm.h"
 #ifdef EXTERNAL_GSM
+#ifdef HAVE_GSM_H
+#include <gsm.h>
+#else
 #include <gsm/gsm.h>
+#endif
 #else
 #include "libgsm/gsm.h"
 #endif
